src/Language/DifferentialDatalog/Debugger/DebugState.hs

```diff
@@ -0,0 +1,179 @@
+{-
+Copyright (c) 2018-2020 VMware, Inc.
```

Comment:
Not sure what the convention is for new files, but usually the year here should be the year it was added (i.e., just 2020)